dcs/action.py

```diff
@@ -1787,6 +1787,25 @@ def dict(self):
         return d
 
 
+class PictureToAll(Action):
+    predicate = "a_out_picture"
+
+    def __init__(self, file_res_key: Optional[ResourceKey] = None):
+        super(PictureToAll, self).__init__(PictureToAll.predicate)
+        if file_res_key:
+            self.file_res_key = file_res_key
+            self.params.append(self.file_res_key)
+
+    @classmethod
+    def create_from_dict(cls, d, mission):
+        return cls(ResourceKey(d["file"]))
+
+    def dict(self):
+        d = super(PictureToAll, self).dict()
+        d["file"] = self.file_res_key.key
```

Comment:
This is wrong. `self.file_res_key` won't be defined by the default constructor. Once you've fixed that, it still might be `None`.